modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobStore.java

```diff
@@ -243,9 +244,14 @@ public void deleteBlobsIgnoringIfNotExists(Iterator<String> blobNames) throws IO
         }
     }
 
-    private void deletePartition(AmazonS3Reference clientReference, List<String> partition, AtomicReference<Exception> aex) {
+    private void deletePartition(
```

Comment:
nit: all other places have `purpose` as first arg, I'd like to keep it consistent, so can we move it to be first here too?

Reply:
Thanks for catching it. It was my intention to keep it consistently but apparently missed this one and the one below. Corrected now.